### ISSUE TITLE
'results in' extensions that would need new ontology terms

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/gocamgen#62]
Here are annotation extension relations used in WB annotations that will require new ontology terms if we want to keep this information but not use the 'results in'-type relations.

I've replaced the term id with the term name to give a better idea of what types of cell or anatomy terms we've recorded for these.

results_in_specification_of(Z4.aaa)

results_in_specification_of(Z1.ppp)

results_in_specification_of(coelomocyte)

results_in_specification_of(AWB)

results_in_morphogenesis_of(hypodermis)

results_in_morphogenesis_of(intestine)

results_in_morphogenesis_of(G1)

results_in_development_of(touch receptor neuron)

results_in_development_of(GABAergic neuron)

results_in_development_of(chemosensory neuron)

results_in_development_of(ALA)

results_in_acquisition_of_features_of(AFD)

We also have some of these chained relations:

regulates_o_results_in_movement_of(Q cell)

regulates_o_results_in_movement_of(Q lineage cell)

regulates_o_results_in_movement_of(DTC)

regulates_o_results_in_movement_of(hermaphrodite distal tip cell)

regulates_o_results_in_movement_of(HSN)

regulates_o_results_in_movement_of(AVM)

regulates_o_results_in_movement_of(QR)

regulates_o_results_in_movement_of(QL)

regulates_o_results_in_development_of(AWB)

**Comment by ukemi** [comment]
I know these were blue in the spreadsheet, but is this sustainable?

**Comment by ukemi** [comment]
In worms I think it is very important to capture these details. Maybe we say those AE relations are ok if the cell ontology doesn't go to the required level of granularity. If not, we come up with some clever way of allowing an annotator to add the specific cell from an approved vocabulary.

**Comment by vanaukenk** [comment]
Yes, in C. elegans, this is critical to capturing the essence of the biology.
But I would argue that, for developmental processes, it is critical regardless of the organism.

**Comment by ukemi** [comment]
Especially as we get to the future complexity of cell types being defined by their expression signatures.
